bindings/java/mongocrypt/src/main/java/com/mongodb/crypt/capi/MongoCryptImpl.java

```diff
@@ -333,6 +316,18 @@ public void close() {
         }
     }
 
+    private void configure(final Supplier<Boolean> successfulConfiguration) {
+        if (!successfulConfiguration.get()) {
+            throwExceptionFromStatus();
+        }
+    }
+
+    private void configure(final Supplier<Boolean> successfulConfiguration, final mongocrypt_ctx_t context) {
```

Comment:
It's not necessarily successful. Rename to just `configuration`?

Reply:
Renamed to `successSupplier`